lib/spack/spack/solver/asp.py

```diff
@@ -4304,6 +4238,9 @@ def _is_reusable(spec: spack.spec.Spec, packages, local: bool) -> bool:
     if "dev_path" in spec.variants:
         return False
 
+    if spec.name == "compiler-wrapper":
+        return False
```

Comment:
Another coupling between core and repo. Maybe this one should be a package property.